README.md

```diff
@@ -1,2 +1,19 @@
-# scirate-bots
-bots for various platforms
+# Scirate-Bots
+Bots for various platforms
+
+# Discord Bot
+### Installation 
```

Comment:
Are you able to write a bit more about what these tokens should be, and how to manage deploying this? I'm not particularly familiar with discord bots, so this would be really helpful!

Reply:
Alrighty, will add those instructions as well